routes/users.js

```diff
@@ -680,8 +719,73 @@ router.post('/applicationchange/accept', function(req, res){
     				};
     			}
     		});
+        var month =monthName(appli.from,appli.to);
```

Comment:
In this one? Didn't we talk about doing it the correct way. Like if I take leave from 15-Jan to 15-Feb. Then 16 holidays for Jan month and 15 holidays for 15 Feb. In the current way it will just add 30/31 leaves to the month of Jan. Northing wrong in this way, just clarification if we are going forward with this?